Branch-SDK/Branch-SDK/BNCDeviceInfo.m

```diff
@@ -124,7 +125,7 @@ + (NSString*) userAgentString {
             BNCPreferenceHelper *preferences = [BNCPreferenceHelper preferenceHelper];
             preferences.browserUserAgentString = browserUserAgentString;
             preferences.lastSystemBuildVersion = self.systemBuildVersion;
-			//NSLog(@"[Branch] userAgentString: '%@'.", browserUserAgentString);
+			BNCLogDebug(@"userAgentString: '%@'.", browserUserAgentString);
```

Comment:
The logging works like this:  The logger always writes the line to the Branch log file (along with data like the date and log level).  The `BNCLogSetLogDisplayLevel` function controls what shows in the console.  When `[Branch setDebug]` is called, it calls `BNCLogSetLogDisplayLevel()` so that all messages are shown in the console log.  Otherwise only warnings and errors are shown in the console.